buildlib/tools/builds.sh

```diff
@@ -201,31 +201,33 @@ build_ugni() {
 # Build CUDA
 #
 build_cuda() {
-	if az_module_load $CUDA_MODULE
-	then
-		if az_module_load $GDRCOPY_MODULE
+	if [ ! -f /opt/rocm/bin/rocminfo ]; then
```

Comment:
maybe check the presence of /usr/lib/cuda or something that indicates we have cuda on the container?
alternatively, define something like "build_cuda: no" for ubuntu2004_rocm job

Reply:
> I followed the existing logic:
> https://github.com/openucx/ucx/blob/master/buildlib/tools/builds.sh#L236
> Should we change there too?

No: 
build_rocm is checking for rocm files
build_cuda should should check for cuda files